docker/logback.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE logback>
+<configuration>
+
+   <appender name="STDOUT" class="ch.qos.logback.core.ConsoleAppender">
+      <encoder>
+         <pattern>%d{HH:mm:ss.SSS} [%thread] %-5level %logger{36} - %msg%n</pattern>
+      </encoder>
+   </appender>
+
+   <root level="INFO">
```

Comment:
I think this should be parameterized as others do so that we can change the log level.
https://github.com/scalar-labs/scalar/blob/master/ledger/conf/log4j.properties.tmpl

Reply:
Right, but at this point I just transferred the `logback.xml` file out of the jar file. To make it a template we need to install Dockerize or a similar templating program. So I thought it's better to do it as an another task.

Reply:
OK, doing it in another PR is fine to me.
I just thought they are pretty related so it is something we can do at once (If we see it as `Dockerfile` refactoring` task).